### PR TITLE
{data}[foss/2016b] ANTs v2.2.0

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.9, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/d1c50a9582a38ac7f3fe0476ffbaaed0 for a full test report.

File: easybuild/easyconfigs/a/ANTs/ANTs-2.2.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+checksums = ['62f8f9ae141cb45025f4bb59277c053acf658d4a3ba868c9e0f609af72e66b4a']
+
+builddependencies = [('CMake', '3.7.1')]
+
+itkver = '4.12.2'
+itkshortver = '.'.join(itkver.split('.')[:2])
+
+dependencies = [
+    ('ITK', itkver, '-Python-2.7.12'),
+    ('VTK', '6.3.0', '-Python-2.7.12'),
```
Comment (boegel) [inline easybuild/easyconfigs/a/ANTs/ANTs-2.2.0-foss-2016b.eb]: @vanzod I would promote `-Python-2.7.12` to a `versionsuffix` that also applies to `ANTs` itself?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2011.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c53e14831f67dcd73296b5900c759d75 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**